mk/plugin.mk

```diff
@@ -22,13 +22,16 @@ else
   DFLAGS :=
 endif
 
-$(PLUGIN): $(SRCS) ../../common/protogen; go build $(DFLAGS) -o $(PLUGIN)
+$(PLUGIN): $(SRCS) ; go build $(DFLAGS) -o $(PLUGIN)
 
 .PHONY: all
-all: $(PLUGIN)
+all: all-hook-pre realall
+
+.PHONY: all-hook-pre
+all-hook-pre:
```

Comment:
Where is the actual command for all-hook-pre ?

Reply:
ditto